### PR TITLE
Fix expiryHours ca-config

### PR DESCRIPTION
**Pull request by Nesze**

### CODE REVIEW

File: examples/config/ca-config [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ca.node.profile=server
 ca.client.profile=client
-ca.endpoint=certificate-authority:8080
\ No newline at end of file
+ca.endpoint=certificate-authority:8080
+ca.serverCert.expiryHours=120
+ca.clientCert.expiryHours=120
```
Comment (Nesze) [inline examples/config/ca-config:5]: Do these look sensible?

File: examples/config/ca-config [hunk 2]
```diff
@@ -1,3 +1,5 @@
 ca.node.profile=server
 ca.client.profile=client
-ca.endpoint=certificate-authority:8080
```
Comment (Nesze) [inline examples/config/ca-config:3]: Any strong opinions on the end of the line change?

File: examples/config/ca-config (hunk 1 above)
Comment (sbuliarca) [reply examples/config/ca-config:5]: If it's not expensive, can we expire them more often?

Comment (Nesze) [reply examples/config/ca-config:5]: Shouldn't be that extensive, should it?

How does setting it to `12h` for start sound?

**Review by MarcinGinszt — APPROVED** [review]
Thanks!

File: examples/config/ca-config (hunk 1 above)
Comment (sbuliarca) [reply examples/config/ca-config:5]: > looks like it is a standard throught the UW

Probably got propagated through copy/pasta

I'd say start with 24h ?

Comment (Nesze) [reply examples/config/ca-config:5]: Sounds like a happy medium.

Comment (Nesze) [reply examples/config/ca-config]: > But I'm not sure whether we should spend this much of effort into it.
indeed

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**